lib/f32.fz

```diff
@@ -149,12 +169,14 @@ f32(val f32) : float is
 
   # equality
```

Comment:
Can you please add a comment that this provides an equivalence relation in the mathematical sense and therefore breaks the IEEE-semantics. `infix =` or `infix !=`  should be used for IEEEE semantics.